.github/workflows/iceoryx.yml

```diff
@@ -2,6 +2,8 @@ name: Build iceoryx
 
 on:
   push:
+    branches:
+      - master
   pull_request:
```

Comment:
I feel we could stick to only build it once per day. For local changes won't affect the result.

Activating this job for PRs would ensure, that we are aware, that  `Iceoryx` master is or is not broken.

Reply:
This is activating the PR job, isn't it? I've just deactivated the `push` job on every PR as I feel it just action the same job as the PR.

Reply:
Yupp it is. I was just suggesting, that we could let it run only once per day. But it actually doesn't matter.